connectors/concur/README.md

```diff
@@ -5,3 +5,19 @@ The Concur connector presents cards inviting the user to approve an expense repo
 Approving an expense report, rejecting an expense report, open the expense report in browser are the actions supported by this connector.
 
 For generic details on how to build, install, and configure connectors, please see the [README](https://github.com/vmware/connectors-workspace-one/blob/master/README.md) at the root of this repository.
+
+
+## Docker
+
+Concur connector requires additional configuration like concur OAuth instance url, client id, and client secret.
+
+```
+docker run --name concur-connector \
+           -p 8080:8080 \
+           ws1connectors/concur-connector \
+           --server.port=8080 \
+           --concur.oauth-instance-url="https://us-impl.api.concursolutions.com" \
+           --concur.client-id="2803b8f8-a42b-43c2-a739-b8768e4759b8" \ 
+           --concur.client-secret="e013335d-b4ce-4c43-a7e4-b67abc1adcb0"
```

Comment:
Forgot a backslash at the end here.

Reply:
When I was doing the change, I was in half sleep :-)
I have raised a PR for adding the backlash. I have pulled the latest image and have verified the command.
https://github.com/vmware/connectors-workspace-one/pull/124